python/src/fast.cpp

```diff
@@ -138,4 +139,138 @@ void init_fast(nb::module_& parent_module) {
         Returns:
             array: The output array.
       )pbdoc");
+
+  m.def(
```

Comment:
I'm wondering if we should have these functions in the Python API? Should we not encourage people to use `mx.quantize` and leave the implementation details hidden? It may be confusing to have two functions to do the same thing..

Or maybe we expose them later if we see that it's useful?